src/locales/en.yaml

```diff
@@ -0,0 +1,44 @@
+username: Username
```

Comment:
I'm hoping for us to use this instead and auto generate JSON once we build for react-intl to consumed. This YAML format will be useful for uploading to Transifex since it supports comment/instruction for our translators.